apps/app-article/src/App.js

```diff
@@ -27,6 +27,7 @@ import ManuallyRelatedArticles from "./components/manually-related-articles";
 import Cookies from "js-cookie";
 import { AndroidView } from "react-device-detect";
 import { Sentry } from "./sentry";
+import DepricationWarning from "./components/deprication-warning";
```

Comment:
sp: deprecation

Reply:
Äsch, I figured that it could be wrong but I never bothered checking. 😅